lib/fbe/octo.rb

```diff
@@ -75,6 +76,11 @@ def Fbe.octo(options: $options, global: $global, loog: $loog)
           methods: [:get],
           backoff_factor: 2
         )
+        builder.use(
+          Fbe::FaradayMiddleware::Quota,
+          logger: loog,
+          github_api_pause: options.github_api_pause || 60
```

Comment:
@Suban05 the argument of `Quota` you better call `pause`, while in the `options` it should be `github_api_pause`. They have different scopes of visibility, that's why different complexity of names.